Gemfile

```diff
@@ -2,6 +2,10 @@ source 'https://rubygems.org'
 
 gemspec
 
+platforms :ruby_19 do
+  gem 'json', '~> 1.8'
+end
+
```

Comment:
@sferik Oh hm, I see it affects Windows as well. Lets just keep it in the Gemfile for now.